Add jq to dev container common toolset

## Problem

The `docs-deploy` composite action in `standard-actions` uses `jq` in its version-command step. On runners where `jq` is not pre-installed, this fails with `jq: command not found` (exit code 127).

This surfaced during the `standard-tooling-plugin` v1.4.5 release: `docs.yml` failed on the `main` branch because the runner lacked `jq`.

```
/__w/_temp/cede73b2-66d4-402f-ba86-9d5cd6a7c723.sh: line 2: jq: command not found
##[error]Process completed with exit code 127.
```

Failed run: https://github.com/wphillipmoore/standard-tooling-plugin/actions/runs/25068304634

## Fix

Add `jq` to the common set of tools installed in the dev container images. It is a reasonable baseline dependency for CI workflows across the fleet.

## Acceptance criteria

- [ ] `jq` is available in the dev container images
- [ ] `docs.yml` succeeds on re-run after the image is updated

--- a/docker/base/Dockerfile.template
+++ b/docker/base/Dockerfile.template
@@ -9,7 +9,7 @@ SHELL ["/bin/bash", "-o", "pipefail", "-c"]
 RUN apt-get update && \
     apt-get upgrade -y && \
     apt-get install -y --no-install-recommends \
-      git curl openssh-client xz-utils gnupg ca-certificates && \
+      git curl jq openssh-client xz-utils gnupg ca-certificates && \
     rm -rf /var/lib/apt/lists/*
 
 # @include common/node-markdownlint.dockerfile

--- a/docker/go/Dockerfile.template
+++ b/docker/go/Dockerfile.template
@@ -8,7 +8,7 @@ SHELL ["/bin/bash", "-o", "pipefail", "-c"]
 RUN apt-get update && \
     apt-get upgrade -y && \
     apt-get install -y --no-install-recommends \
-      git openssh-client xz-utils gnupg ca-certificates && \
+      git jq openssh-client xz-utils gnupg ca-certificates && \
     rm -rf /var/lib/apt/lists/*
 
 # @include common/node-markdownlint.dockerfile

--- a/docker/java/Dockerfile.template
+++ b/docker/java/Dockerfile.template
@@ -8,7 +8,7 @@ SHELL ["/bin/bash", "-o", "pipefail", "-c"]
 RUN apt-get update && \
     apt-get upgrade -y && \
     apt-get install -y --no-install-recommends \
-      git curl openssh-client xz-utils gnupg ca-certificates && \
+      git curl jq openssh-client xz-utils gnupg ca-certificates && \
     rm -rf /var/lib/apt/lists/*
 
 # @include common/node-markdownlint.dockerfile

--- a/docker/python/Dockerfile.template
+++ b/docker/python/Dockerfile.template
@@ -8,7 +8,7 @@ SHELL ["/bin/bash", "-o", "pipefail", "-c"]
 RUN apt-get update && \
     apt-get upgrade -y && \
     apt-get install -y --no-install-recommends \
-      git curl openssh-client xz-utils gnupg ca-certificates && \
+      git curl jq openssh-client xz-utils gnupg ca-certificates && \
     rm -rf /var/lib/apt/lists/*
 
 # @include common/node-markdownlint.dockerfile

--- a/docker/ruby/Dockerfile.template
+++ b/docker/ruby/Dockerfile.template
@@ -8,7 +8,7 @@ SHELL ["/bin/bash", "-o", "pipefail", "-c"]
 RUN apt-get update && \
     apt-get upgrade -y && \
     apt-get install -y --no-install-recommends \
-      build-essential git curl openssh-client xz-utils gnupg ca-certificates && \
+      build-essential git curl jq openssh-client xz-utils gnupg ca-certificates && \
     rm -rf /var/lib/apt/lists/*
 
 # @include common/node-markdownlint.dockerfile

--- a/docker/rust/Dockerfile.template
+++ b/docker/rust/Dockerfile.template
@@ -8,7 +8,7 @@ SHELL ["/bin/bash", "-o", "pipefail", "-c"]
 RUN apt-get update && \
     apt-get upgrade -y && \
     apt-get install -y --no-install-recommends \
-      git curl openssh-client pkg-config xz-utils gnupg ca-certificates && \
+      git curl jq openssh-client pkg-config xz-utils gnupg ca-certificates && \
     rm -rf /var/lib/apt/lists/*
 
 # @include common/node-markdownlint.dockerfile

